test/EventStore.Client.Tests.Common/EventStoreTestServer.cs

```diff
@@ -64,6 +64,8 @@ public class EventStoreTestServer : IEventStoreTestServer {
 				.WithName(ContainerName)
 				.MountVolume(_hostCertificatePath, "/etc/eventstore/certs", MountType.ReadOnly)
 				.ExposePort(2113, 2113)
+				//.KeepContainer()
+				//.KeepRunning()
```

Comment:
Do we need to keep these comments?

Reply:
we dont need to but i did keep them on purpose, uncommenting those will leave the container running after a test if you want to, say, open the eventstore ui and look at something manually

Reply:
Makes sense, thanks!